data.json

```
@@ -1805,6 +1805,15 @@
             ],
             "description": "Application to comfortably monitor network traffic."
         },
+	{
+            "name": "Sugar",
+            "link": "https://github.com/KilroyWasHere-cs-j/Sugar.git",
```

```suggestion
            "link": "https://github.com/KilroyWasHere-cs-j/Sugar",
```
The `.git` extension is not needed when linking to the repository webpage.